airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -635,6 +639,8 @@ def _start_task(self):
             run_opts["tags"] = [{"key": k, "value": v} for (k, v) in self.tags.items()]
         if self.propagate_tags is not None:
             run_opts["propagateTags"] = self.propagate_tags
+        if self.count is not None:
+            run_opts["count"] = count
```

Comment:
It will raise 

```console
Traceback (most recent call last):
  File "/opt/airflow/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/opt/airflow/airflow/providers/amazon/aws/operators/ecs.py", line 549, in execute
    self._start_task()
  File "/opt/airflow/airflow/providers/amazon/aws/operators/ecs.py", line 643, in _start_task
    run_opts["count"] = count
NameError: name 'count' is not defined
```